platform-includes/session-replay/setup-canvas/javascript.astro.mdx

```diff
@@ -1,17 +0,0 @@
-```javascript {13} {filename:sentry.client.config.js}
```

Comment:
m: These filenames are important, I don't think we can just remove these and have the blocks generic :(

Reply:
hmm, maybe we can add a frontmatter bit with the filename somewhere?

Reply:
You are right, I missed that. That's actually bad since this will force us again to create separate files.

@a-hariti adding config file names would get messy since some frameworks have up to three different config files depending on the runtime

Reply:
I'm not sure if the need for this feature is actually worth the time, maybe we'll just close it and not overcomplicate this here – wdyt?

Reply:
I agree

Reply:
Agreed on closing